### DISCUSSION TITLE
(enter) in msg

### DISCUSSION THREAD

**Discussion by gibscarter** [FabCx/TradingView-Webhook-Bot#44] [General]
Can i write msg with (enter) in it?
Because i might fill msg with =  
"msg": "{{strategy.order.comment}}"

And the \n in pine will automatically convert into (enter) when the final alert came, any suggestion?

Thank you!

**Comment by gibscarter** [comment]
Any other command i can use beside \n to declare enter?
That tradingview cant read, but python can?

**Reply by fabston** [reply]
Hi,

Since the bot processes alerts in a json string only `\n` will work. A line break will result in errors.

**Reply by gibscarter** [reply]
Do u have any suggestion to do so? Or update the rules so people can send alerts with \n inside the pine without any issue?

Previous version was fine with that, but i kinda need to send it to different channel which is only available on current version:)
Anyway thanks for responding

**Reply by fabston** [reply]
Give this a try `\\n`

**Reply by gibscarter** [reply]
Hey it seems give the \n output in the comment, 
Thanks, really appreciate it!

**Reply by fabston** [reply]
That's what I expected to happen 😉

**Comment by danywayGit** [comment]
Thank you, this tips helped me.
